centreon/www/class/centreon-clapi/centreon.Config.Poller.class.php

```
@@ -570,6 +570,31 @@ public function cfgMove($variables = null)
+                }
+            }
+
+            // Restart centreon_vmware on central only if VMWare config has changed
+            $vmwareStatement = $pearDB->prepare(
+                'SELECT vmware_updated FROM nagios_server WHERE id = :pollerId'
+            );
+            $vmwareStatement->bindValue(':pollerId', (int) $pollerId, PDO::PARAM_INT);
+            $vmwareStatement->execute();
+            $vmwareRow = $vmwareStatement->fetch(PDO::FETCH_ASSOC);
+            if ($vmwareRow && $vmwareRow['vmware_updated'] === '1') {
+                exec(escapeshellcmd('sudo -n -- systemctl restart centreon_vmware'), $restartOutput, $restartReturnCode);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$restartOutput'. (undefined)

(UnusedLocalVariable)